### ISSUE TITLE
Pre-training-Using-Knowledge-Distillation is better than Pre-training-Only for downstream tasks?

### ISSUE THREAD

**Issue by guotong1988** [alexa/bort#10]
![image](https://user-images.githubusercontent.com/4702353/109586855-03b1b280-7b41-11eb-9b94-c28038cf8bec.png)

But this picture shows the same MLM accuracy to the end. 
What is the downstream tasks' performance comparison for pre-training-using-knowledge-distillation and pre-training-only?
I am new to BORT. Thank you very much. @adewynter

![image](https://user-images.githubusercontent.com/4702353/109589154-df57d500-7b44-11eb-9e25-2ea475793ac0.png)

**Comment by adewynter** [comment]
Hi!

So the accuracy is indeed the same when talking about convergence points. However, the _speed_ at which they reach it is significantly different. That is, at some `t`, `error_KD > error_self-supervised`, even when for some `T >> t`, `error_KD = error_self-supervised`.

As an example, think of the convergence of, say, GD for a smooth, convex function -- the error at some step `t` would be something like `O(1/t^2)`. For this architecture, at some `t` the MLM accuracy of KD was significantly higher than self-supervised, even when eventually they both reached the same convergence point.

I suppose that paragraph could be phrased slightly better, though, so thanks for bringing that up!

-Adrian

PS: One more thing -- we did not attach in the paper experiments for _downstream_ KD vs self-supervised performance, because the algorithm we used (Agora) was much better than these methods. The plot above is for pre-training!
